docs/_config.yml

```diff
@@ -7,9 +7,9 @@ baseurl: /java-slack-sdk
 url: https://slack.dev
 
 sdkLatestVersion: 1.1.5
-okhttpVersion: 4.8.1
+okhttpVersion: 4.9.0
```

Comment:
Even though merging this PR affects the live document, I think it's safe enough to reflect this version with this SDK's v1.1 series in docs.